local-build.sh

```diff
@@ -36,8 +36,8 @@ function node_version() {
   if [ -r ${NV_FILE} ]; then
     NODE_VERSION=$(<"$NV_FILE")
   fi
-  # docusausrus 2.0.0-beta.18 and later ony work with node 16.14 later, use it as the default
-  NODE_VERSION=${NODE_VERSION:-16.14}
+  # docusausrus 2.0.0-beta.18 and later only work with node 18.17 later, use it as the default
```

Comment:
Rather than an incorrect comment, how about: "Set the node version to use (LTS)"